### PR TITLE
Adding Freyja 1.3.1

### PR DESCRIPTION
**Pull request by kevinlibuit**
- [x] This comment contains a description of what is in the pull request.
Adding Freyja 1.3

This PR is to add a new version of the Freyja tool that already has dockerfiles for previous versions
- [x] Build your own docker image using a Dockerfile
  - [x] Directory structure should be name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. spades/3.12.0/Dockerfile)
  - [x] Includes the recommended LABELS
- [x] (Optional) Dockerfile is built with best practices and has been approved by a linter (such as https://hadolint.github.io/hadolint/)
- [x] Edit main README.md
- [x] Ensure tool is listed in Program_Licenses.md
- [ ] Create a simple container-specific README.md in the same directory as the Dockerfile (i.e. spades/3.12.0/README.md)
- [ ] Update GitHub actions workflow if needed
- [x] Any files required for building are located in the same directory as the Dockerfile (i.e. spades/3.12.0/my_spades_tests.sh)
- [x] Have successfully run the workflow "Test <program name> image" in your forked repository

### CODE REVIEW

**Comment by kevinlibuit** [comment]
Please ignore! Freyja 1.3.1 coming soon with useful updates. Will open a separate PR once v1.3.1 becomes available

**Comment by kevinlibuit** [comment]
Reopenning for v1.3.1

**Comment by kevinlibuit** [comment]
This PR is to add a new version (v1.3.1) of the Freyja tool that already has dockerfiles for previous versions
- [x] Build your own docker image using a Dockerfile
  - [x] Directory structure should be name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. spades/3.12.0/Dockerfile)
  - [x] Includes the recommended LABELS
- [x] (Optional) Dockerfile is built with best practices and has been approved by a linter (such as https://hadolint.github.io/hadolint/)
- [x] Edit main README.md
- [x] Ensure tool is listed in Program_Licenses.md
- [ ] Create a simple container-specific README.md in the same directory as the Dockerfile (i.e. spades/3.12.0/README.md)
- [ ] Update GitHub actions workflow if needed
- [x] Any files required for building are located in the same directory as the Dockerfile (i.e. spades/3.12.0/my_spades_tests.sh)
- [x] Have successfully run the workflow "Test <program name> image" in your forked repository

File: freyja/1.3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+LABEL software="Freyja"
+LABEL software.version=${FREYJA_SOFTWARE_VERSION}
+LABEL description="Freyja is a tool to recover relative lineage abundances from mixed SARS-CoV-2 samples from a sequencing dataset (BAM aligned to the Hu-1 reference)"
+LABEL website="https://github.com/andersen-lab/Freyja"
+LABEL license="https://github.com/andersen-lab/Freyja/blob/main/LICENSE"
+LABEL maintainer="Kevin Libuit"
+LABEL maintainer.email="kevin.libuit@theiagen.com"
+
+# Create Freyja conda environment and add to PATH
+RUN conda create -n freyja-env -c defaults -c bioconda -c conda-forge freyja=${FREYJA_SOFTWARE_VERSION} \
```
Comment (kapsakcj) [inline freyja/1.3/Dockerfile:17]: It's strongly recommended by bioconda to set the channels in this specific order:
conda-forge - top priority
bioconda
defaults - lowest priority

The way you have it set right now is in opposite order.

I recommend changing this line to:
```
conda create -n freyja-env -c conda-forge -c bioconda -c defaults freyja=${FREYJA_SOFTWARE_VERSION}
```

I'm getting this message while building and it's taking a looong time (>5min) to solve the environment
```
Step 12/14 : RUN conda create -n freyja-env -c defaults -c bioconda -c conda-forge freyja=${FREYJA_SOFTWARE_VERSION} && echo "source activate freyja-env" > ~/.bashrc
 ---> Running in 6228ea900e9d
Collecting package metadata (current_repodata.json): ...working... done
Solving environment: ...working... failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working...

```

Comment (kapsakcj) [reply freyja/1.3/Dockerfile:17]: oops, I meant to comment ^ on the 1.3.1 dockerfile, not 1.3. The same still applies though

**Comment by kapsakcj** [comment]
When I switched the conda install step to this:
```
conda create -n freyja-env -c conda-forge -c bioconda -c defaults freyja=${FREYJA_SOFTWARE_VERSION}
```
it reduced the build time by 4 minutes
```
Successfully built 27739a19624a
Successfully tagged kevin/frejya:1.3.1

real    8m0.424s
user    0m0.047s
sys     0m0.135s
```
Still got the failure to solve environment warning, but it eventually solved
```
Collecting package metadata (current_repodata.json): ...working... done
Solving environment: ...working... failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done
```

**Comment by kevinlibuit** [comment]
Great catch, @kapsakcj! Switched the order based on priority in most recent commit

**Comment by kapsakcj** [comment]
OK, I fixed up the README and tested out the build of the latest version of the 1.3.1 dockerfile. Builds successfully in about 8 min:
```
Successfully built c77dddb993f0
Successfully tagged kevin/freyja:1.3.1

real    8m20.683s
user    0m0.118s
sys     0m0.077s
```

I'll go ahead and merge and build on dockerhub & quay 👍

**Pull request merged**